.github/release-drafter.yml

```diff
@@ -0,0 +1,45 @@
+name-template: 'v$NEXT_MINOR_VERSION'
+tag-template: 'v$NEXT_MINOR_VERSION'
+categories:
+  - title: '🚀 Features'
+    labels:
+      - "feature"
+      - "enhancement"
+  - title: '💣 Breaking Change'
+    labels:
+      - "breaking"
+      - "change"
+  - title: '🐛 Bug Fixes'
+    labels:
+      - "fix"
+      - "bug"
+      - "bugfix"
+  - title: '📝 Documentation'
+    labels:
+      - "documentation"
+      - "docs"
+  - title: '🔨 Maintenance'
+    labels:
+      - "chore"
+      - "dependencies"
+      - "ci"
+change-template: '- $TITLE @$AUTHOR (#$NUMBER)'
+template: |
+  *Help make the NGINX Ingress Controller better by participating in our [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=L_093Ttq0UCb4L-DJ9gcUM6Dh1A0cORCorfgZAMdkwJUREhJUFAyM1ZHRzZLSzQyMUlCNFhXVkZENy4u)!*
+
+  ## New in NGINX Ingress Controller v$NEXT_MINOR_VERSION
+
+  $CHANGES
+
+  ## Upgrade
+
+  For NGINX, use the v$NEXT_MINOR_VERSION image from our DockerHub: `nginx/nginx-ingress:$NEXT_MINOR_VERSION`, `nginx/nginx-ingress:$NEXT_MINOR_VERSION-alpine` or `nginx/nginx-ingress:$NEXT_MINOR_VERSION-ubi`
+  For NGINX Plus, please build your own image using the v$NEXT_MINOR_VERSION source code.
+  For Helm, use version 0.7.1 of the chart.
```

Comment:
Is there a way to use a variable here instead of `0.7.1`? If not, I'd take out `0.7.1` and put in something like `HELM_VERSION_REPLACE_ME!`

Reply:
yeah I think we would need to use `HELM_VERSION_REPLACE_ME!`...or do we even have to mention helm here? 🤔

Reply:
we mention that because we release helm chart as part of a IC release